config/rails.yml

```diff
@@ -0,0 +1,21 @@
+Rails:
+  Enabled: true
+
+Rails/FindEach:
+  Enabled: false
+
+Rails/OutputSafety:
+  Enabled: true
+
+Rails/PluralizationGrammar:
+  Enabled: true
+
+Rails/RequestReferer:
+  Enabled: true
+  EnforcedStyle: referrer
+
+Rails/ScopeArgs:
+  Enabled: true
+
+Rails/UniqBeforePluck:
+  Enabled: true
```

Comment:
`GitHub/RailsApplicationRecord` seems like a reasonable compromise for now. I am tempted to try to patch this upstream as well.